src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
@natefaubion the diff is appearing funny - it's a comment that was added to this line that I'm commenting on, the code is unchanged.